data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py

```diff
@@ -217,7 +247,28 @@ def get_query_specs(self, *args, **keyword_args) -> query_spec_list:
                         field['name'] for field in fields_for(OBSERVATION)))
         }
 
-        return [sandbox_query_dict, delete_query_dict, insert_query_dict]
+        insert_mapping_query_dict = {
+            cdr_consts.QUERY:
+                INSERT_MAPPING_QUERY.render(
+                    project=self.project_id,
+                    sandbox_dataset=self.sandbox_dataset_id,
+                    sandbox_table=self.sandbox_table_for(OBSERVATION),
+                    dataset=self.dataset_id)
+        }
+
+        delete_mapping_query_dict = {
+            cdr_consts.QUERY:
+                DELETE_MAPPING_QUERY.render(
+                    project=self.project_id,
+                    sandbox_dataset=self.sandbox_dataset_id,
+                    sandbox_table=self.sandbox_table_for(OBSERVATION),
+                    dataset=self.dataset_id)
+        }
+
+        return [
+            sandbox_query_dict, delete_query_dict, insert_query_dict,
+            insert_mapping_query_dict, delete_mapping_query_dict
```

Comment:
I would suggest moving `delete_query_dict` to later, maybe before `delete_mapping_query_dict`